subfunctions/demo_CaBBI_QGIF

```diff
@@ -0,0 +1,232 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
What I meant is that the demo should be a function, ie starting with:
`function demo_CaBBI_QGIF to ensure workspace isolation